src/app/services/project_service.js

```diff
@@ -138,18 +138,30 @@ angular
             _.each(service.files.manifest.sources, function(node) {
                 node.label = "" + node.source_name + "." + node.name;
                 service.files.manifest.nodes[node.unique_id] = node;
+                node.color = '#5fb825'
```

Comment:
just porting over the selectors that we deleted above. There are better ways to do this - just sharing as an example!